jobboard/templates/jobboard/job.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %} {% block content %}
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md p-5 text-align">
+            <h3>{{ job_data.title }} at {{ job_data.company }}</h3>
+            <h5>{{ job_data.description }}</h5>
```

Comment:
I think this page needs to include all of the data about the job, it doesn't make sense that you click on "read more" but then fewer details are visible than before.

Reply:
Yeah, you are right. 
Will add more details